xml/installation-installation-ses_integration.xml

```diff
@@ -376,4 +376,49 @@ rgw keystone verify ssl = false # If keystone is using self-signed
     </step>
    </procedure>
   </section>
+    <section>
+   <title>Enable Copy-On-Write Cloning of Images</title>
+   <para>
+    Due to a security issue described in <link
+    xlink:href="http://docs.ceph.com/docs/master/rbd/rbd-openstack/?highlight=uuid#enable-copy-on-write-cloning-of-images"
+    />, the copy-on-write cloning of images when
+    &o_img; and &o_blockstore; are both using a &ceph; back-end is not
+    supported. However, if you want to use this feature for faster operation,
```

Comment:
&o_img; and &o_blockstore; are both using a &ceph;  back-end is not supported. <- this doesn't make sense? Is there something missing here?

Reply:
I dunno. That's what the description said. Maybe @bryanstephenson can shed some light on it.

Reply:
Due to a security issue described in ... , the COW cloning of images is not supported when Glance and Cinder are both using a Ceph back-end.

Reply:
Makes sense to me, thanks Carl.

Reply:
It makes sense to me. Maybe say it this way instead:

the copy-on-write cloning of images when both
    &o_img; and &o_blockstore; are using a &ceph; back-end is not
    supported.